src/theme/DocItem/Footer/index.js

```diff
@@ -0,0 +1,147 @@
+import React, { useState, useRef } from 'react'
+import clsx from 'clsx'
+import Footer from '@theme-original/DocItem/Footer'
+import { useDoc } from '@docusaurus/theme-common/internal'
+import styles from './styles.module.css'
+import { trackStructEvent } from '@snowplow/browser-tracker'
+
+function CommentBox({ handleSubmit, feedbackTextRef }) {
+  const placeholder = "Why wasn't it helpful?"
+
+  return (
+    <div className="row">
+      <form onSubmit={handleSubmit}>
+        <textarea
+          className={clsx(styles.commentBox, 'col')}
+          ref={feedbackTextRef}
+          placeholder={placeholder}
+          rows={3}
+          cols={34}
+          maxLength="999"
+        />
+        <button
+          className={clsx(
+            'navbar__item',
+            'snwpl-nav-button',
+            'col',
+            styles.feedbackButton,
+            styles.submitButton
+          )}
+          type="submit"
+        >
+          Send feedback
+        </button>
+      </form>
+    </div>
+  )
+}
+
+function Feedback() {
+  const { permalink } = useDoc().metadata
+  const feedbackTextRef = useRef()
+  const buttonLikeRef = useRef()
+  const buttonDislikeRef = useRef()
+
+  const [isTextboxVisible, setIsTextboxVisible] = useState(false)
+  const [isThanksVisible, setIsThanksVisible] = useState(false)
+
+  const handleLike = () => {
+    buttonLikeRef.current.blur()
+    setIsTextboxVisible(false)
+
+    setIsThanksVisible(true)
+    setTimeout(() => {
+      setIsThanksVisible(false)
+    }, 1000)
+
+    trackStructEvent({
+      category: 'feedback',
+      action: 'like',
+      label: permalink,
+    })
+  }
+
+  const handleDislike = () => {
+    buttonDislikeRef.current.blur()
+    setIsTextboxVisible((current) => !current)
+    setIsThanksVisible(false)
+
+    trackStructEvent({
+      category: 'feedback',
+      action: 'dislike',
+      label: permalink,
+    })
+  }
+
+  const handleSubmit = (event) => {
+    event.preventDefault()
+    const text = feedbackTextRef.current.defaultValue
+
+    trackStructEvent({
+      category: 'feedback',
+      action: 'comment',
+      label: permalink,
+      property: text,
+    })
+
+    setIsTextboxVisible((current) => !current)
+    setIsThanksVisible(true)
+    setTimeout(() => {
+      setIsThanksVisible(false)
+    }, 1000)
+  }
+
+  return (
+    <div className={clsx('col', styles.feedback)}>
+      <div className={clsx('row', styles.feedback)}>
+        <div className={clsx('row', styles.feedbackQuestion)}>
+          Was this page helpful?
```

Comment:
See if you have the spare minute to add this
![image](https://user-images.githubusercontent.com/2670454/223798958-f1ef9854-f1b9-4c64-88ad-ab2546fdf0bd.png)
as discussed on Slack